### PR TITLE
Zoom installation docs update

### PR DESCRIPTION
**Pull request by justinegeffen**
This PR removes reverences to a previously removed 'enable password based authentication' toggle in the setup documentation. It also replaces the image of the settings. Fixes: https://github.com/mattermost/mattermost-plugin-zoom/issues/229

### CODE REVIEW

File: docs/installation/install-zoom-plugin.md [hunk 1]
```diff
@@ -4,19 +4,20 @@ description: Get the Mattermost plugin installed as a first step
...
 
-If your server doesn't have internet access, you can download the [latest plugin binary release](https://github.com/mattermost/mattermost-plugin-zoom/releases) and upload it to your server via **System Console &gt; Plugin Management**. The binary releases on the page above, are the same as used by the Plugin Marketplace.
-
-1. Go to **Main Menu &gt; Plugin Marketplace** in Mattermost.
-2. Search for "Zoom" or manually find the plugin from the list and click **Install**.
-3. After the plugin has downloaded and been installed, follow the instructions on [Zoom Configuration](zoom-configuration/).
-4. After finishing the [Zoom Configuration](zoom-configuration/), follow the instructions on [Mattermost Setup](mattermost-setup.md).
+If your server doesn't have internet access, you can download the [latest plugin binary release](https://github.com/mattermost/mattermost-plugin-zoom/releases) and upload it to your server via **System Console > Plugin Management**. The binary releases on the page above, are the same as used by the Marketplace.
 
+1. Go to **Main Menu > Marketplace** in Mattermost.
```
Comment (larkox) [inline docs/installation/install-zoom-plugin.md:18]: Is this correct? Shouldn't this be something like "Go to the system console, manage plugins"?

File: docs/installation/zoom-configuration/README.md [hunk 2]
```diff
@@ -6,21 +6,21 @@ description: >-
...
+Zoom version 1.5 supports one authentication method for users to connect Mattermost and Zoom: OAuth.
+
+* There are two types of OAuth Zoom apps you can create - **Account** or **User** level apps. You can use either one with this Zoom plugin depending on your organization's security and UX preferences.
+
+  * **Account-level app**
+    * Users don't need to authorize their Zoom accounts before starting their first Zoom meeting. The only requirement is that their Mattermost account uses the same email address as their Zoom account.
+    * Users can't connect their Mattermost/Zoom accounts if their emails do not match.
+  * **User-level app**
+    * Each user needs to connect their Zoom account with their Mattermost account before they can use the integration. When they try to create a meeting for the first time, they'll receive a message to connect their account, and will need to click **Approve** on the pop-up confirmation notice.
```
Comment (larkox) [inline docs/installation/zoom-configuration/README.md:17]: Not sure if worth it, but the actual flow is:
- They try to create a meeting
- They must follow a link
- The link will redirect to a zoom owned website
- On this site, if they are not logged in, they will be asked to log in into zoom
- Once they are logged in, they will see the "approve" button.

Probably with what is already written, everything is clear, but wanted to make sure.

File: docs/installation/zoom-configuration/webhook-configuration.md [hunk 3]
```diff
@@ -1,21 +1,20 @@
...
-3. Click **Add New Event Subscription** and give it a name \(e.g. "Meeting Ended"\).
+3. Select **Add New Event Subscription** and give it a name \(e.g. "Meeting Ended"\).
 4. Enter a valid **Event notification endpoint URL** \(`https://SITEURL/plugins/zoom/webhook?secret=WEBHOOKSECRET`\).
    * `SITEURL` should be your Mattermost server URL.
    * `WEBHOOKSECRET` is generated during [Mattermost Setup](../mattermost-setup.md).
 
 ![Feature screen](../../.gitbook/assets/screenshot-from-2020-06-05-19-51-56%20%284%29.png)
 
-* Click **Add events** and select the **End Meeting** event.
+* Select **Add events** and select the **End Meeting** event.
```
Comment (larkox) [inline docs/installation/zoom-configuration/webhook-configuration.md:16]: Should these be bullet points or numbers?

File: docs/installation/zoom-configuration/webhook-configuration.md [hunk 4]
```diff
@@ -1,21 +1,20 @@
...
 
 ![Event types screen](../../.gitbook/assets/screenshot-from-2020-06-05-20-43-04%20%282%29%20%281%29.png)
 
-* Click **Done** and then save your app.
-
+  * Select **Done** and then save your app.
```
Comment (larkox) [inline docs/installation/zoom-configuration/webhook-configuration.md:20]: Are the two spaces before this line intended?

**Review by larkox — APPROVED** [review]
Sorry for the delay. LGTM, just small comments, but nothing blocking.

File: docs/installation/install-zoom-plugin.md (hunk 1 above)
Comment (justinegeffen) [inline docs/installation/install-zoom-plugin.md:18]:
```suggestion
1. Go to **System Console > Manage Plugins** in Mattermost.
```

**Pull request merged**